Tasks/Src/RTOS_BPS.c

```diff
@@ -4,9 +4,20 @@ RTOS_BPS library includes all the wrapper functions for a functioning RTOS - nee
 
 #include "RTOS_BPS.h"
 
-void RTOS_BPS_SemPend(void) {
+/**
+ * @brief Pends a BPS_OS_Semaphore.
+ * @param *sem - pointer to a sempaphore to pend
+ * @param tick - time in clock ticks to timeout for
+ * @param opt - pend option
+ * @return the semaphore count, or 0 if not available
+ */
```

Comment:
Change the comment headers to match the input parameters.

Reply:
wdym, they do match the input parameters unless i am just blind
![image](https://user-images.githubusercontent.com/107018918/192367328-a64ac9eb-5500-4ec7-addd-c5ba2b783573.png)

Reply:
No, I'm blind